app/AppLoader.scala

```diff
@@ -29,6 +30,15 @@ class AppLoader extends ApplicationLoader with Logging {
 
     val extraConfig: Configuration = extraConfigs.foldRight(Configuration.empty)(_.configuration(context.environment.mode).withFallback(_))
 
+    val role: Option[String] = extraConfig.getOptional[String]("accounts.LoggingRole")
```

Comment:
As identified in the related cloudformation PR, the role isn't needed as the stream is in the same account as the running application and we can remove this val.